### PR TITLE
Initial gen7 documentation

### PR DESCRIPTION
**Pull request by TheSonAlsoRises**
Super rough documentation for generation 7.

* ~~Classes are missing pretty much everywhere~~
* Code is probably horribly indented
* Everything was stolen from my SVExchange documentation

### CODE REVIEW

File: client/static/static.module.js [hunk 1]
```diff
@@ -3,12 +3,16 @@ ng.module('porybox.static', ['ngRoute']).config(['$routeProvider', $routeProvide
   [
     'about',
     'donate',
-    'extracting-pk6-files',
+    'extracting-pokemon-files',
```
Comment (not-an-aardvark) [inline client/static/static.module.js]: I think we should set up a redirect here so that links to the old page aren't broken.

Comment (Raia) [inline client/static/static.module.js]: Oh yeah, that's probably a good idea.

File: client/static/extracting-pokemon-files.html [hunk 2]
```diff
@@ -83,6 +83,61 @@ <h1 class="md-display-2">Extracting Pokémon from your game</h1>
...
+        <td>SD card reader</td>
+        <td>No</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>4</td>
+        <td><a href="/#/how-to-pk7-4-tea">TEA checking with KeySAVe</a></td>
+        <td>Digital/retail</td>
+        <td>CFW, NTR CFW</td>
+        <td>=<11.2.0</td>
```
Comment (not-an-aardvark) [inline client/static/extracting-pokemon-files.html]: I think this is malformed html -- to use a `<` symbol in an html page, you should use `&lt;`.

Comment (TheSonAlsoRises) [inline client/static/extracting-pokemon-files.html]: Fixed, thanks.

**Pull request merged**